glean-core/ios/GleanTests/Debug/GleanDebugUtilityTests.swift

```diff
@@ -20,38 +20,37 @@ class GleanDebugUtilityTests: XCTestCase {
     }
 
     func testHandleCustomUrlLogPings() {
-        // Test initial state
-        XCTAssertFalse(Glean.shared.configuration!.logPings)
+        // We destroy the Glean handle so that Glean in in an unitialized state,
+        // this way it will save the value of `logPings` in the `logPings` prop.
+        Glean.shared.testDestroyGleanHandle()
 
         // Test toggle true
         var url = URL(string: "test://glean?logPings=true")
         Glean.shared.handleCustomUrl(url: url!)
-        XCTAssertTrue(Glean.shared.configuration!.logPings)
+        XCTAssertTrue(Glean.shared.logPings!)
```

Comment:
If `logPings` weren't Optional, you wouldn't need the `!` (force-unwrap) operator here, which could cause a runtime exception in tests if `logPings` were `nil`